rich-default/environment.yml

```diff
@@ -0,0 +1,522 @@
+name: base
+channels:
+  - samoturk
+  - mordred-descriptor
+  - rdkit
+  - conda-forge
+  - defaults
```

Comment:
If you have to have these other channels I think that this is better:

- conda-forge
- defaults
- other packages

then set channel_priority_strict=True.

Reply:
This would pull everything it can from conda forge, then look for any missing packages in defaults, and then so-on. You want to maximize what is installed from the self-consistent channels (conda-forge and defaults).

Reply:
It currently gives an error if I set the channel priority to be true (probably due to differences in dependencies between packages). I will request to merge this PR beforehand, and later examine the packages more thoroughly and try to submit to conda-forge in another issue/PR.